services/github/github-hacktoberfest.tester.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+const isHactoberfestCombinedStatus = Joi.string().regex(
+  /^[0-9]+ open issues(, [0-9]+ PRs)?(, [0-9]+ days left)?$/
```

Comment:
Haven't actually tested this, but just a tweak to handle the singular form of `issues` and/or `days`
```suggestion
  /^[0-9]+ open issues?(, [0-9]+ PRs)?(, [0-9]+ days? left)?$/
```

Reply:
Doesn't `PRs` also need the switch on it?

Reply:
Updated, thanks!